specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2023-01-02/disk.json

```diff
@@ -467,6 +467,55 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}/disks/{diskName}/beginGetAccess": {
```

Comment:
>beginGetAccess [](http://example.com/codeflow?start=152&length=14)

how does one create this resource? I dont see a PUT or a get on /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}/disks/{diskName}

Reply:
"virtualMachineScaleSets/disks" is a modelled as a child resourcetype as "ProxyOnly". This resourceType refers to a compute disk of a virtualmachinescaleset's vminstance. These disks are not modelled as tracked resource in ARM (hence: no PUT or REGISTER action) and completely platform managed. The above RT is a way to lookup a property i.e. disk's SAS URI leveraging ARM RBAC and validation with the intent to be invoked by an allowed internal RP only.